fiorella/src/styles/layout/_header.scss

```diff
@@ -0,0 +1,304 @@
+/*================ Site Header ================*/
+.homepage-logo {
+  display: none;
+}
+
+.navbar {
+  display: none;
+}
+
+.desktop-btn-search {
+  display: none;
+}
+
+.header-buttons-container {
+  display: none;
+}
+
+.menu-container {
+  background-color: $color-white-solid;
+  font-family: $font-menu-header;
+  height: 100%;
+  overflow-x: hidden;
+  position: fixed;
+  transition: 0.5s;
+  width: 0;
+  z-index: 2;
+
+  a {
+    color: $color-black-solid;
+    font-size: 25px;
+    text-decoration: none;
+  }
+
+  @include reset-ul(block);
+}
+
+.show-menu {
+  padding: 0 20px;
+  width: 100%;
+}
+
+.menu-top-section {
+  display: flex;
+  justify-content: center;
+  padding-top: 10px;
+  position: relative;
+
+  .menu-close {
+    @extend %positioning;
+    justify-content: flex-end;
+  }
+
+  button {
+    background-color: transparent;
+    border-style: none;
+  }
+}
+
+.search-container {
+  border-bottom-color: $color-featured;
+  border-bottom-style: solid;
+  margin: 20px 0 5px;
+  padding-bottom: 5px;
+
+  form {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .desktop-btn-search {
+    display: none;
+  }
+
+  button {
+    @extend %transparent-button;
+    padding-right: 0;
+  }
+
+  input {
+    background-color: transparent;
+    border-style: none;
+    font-size: 15px;
+    text-transform: uppercase;
+  }
+
+  ::placeholder {
+    color: $color-black-solid;
+    opacity: 1;
+  }
+
+  :-ms-input-placeholder {
+    color: $color-black-solid;
+  }
+
+  ::-ms-input-placeholder {
+    color: $color-black-solid;
+  }
+}
+
+.mobile-nav {
+  border-bottom-color: $color-black-solid;
+  border-bottom-style: solid;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+  padding-bottom: 10px;
+
+  a {
+    text-transform: uppercase;
+  }
+}
+
+.mobile-user-buttons {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+
+  a {
+    color: $color-featured;
+  }
+}
+
+.mobile-info-buttons {
+  margin-top: 10px;
+}
+
+.mobile-top-section {
+  display: flex;
```

Comment:
Properties should be ordered align-items, background-color, display, height, justify-content, position